server/src/main/java/org/jboss/as/server/BootstrapImpl.java

```diff
@@ -214,8 +214,9 @@ void done() {
             setResult(container);
         }
 
-        void failed(Throwable t) {
-            setFailed(t);
+        void failed(final Throwable t) {
```

Comment:
Could you add a Java doc describing the argument cannot be null?

Reply:
Fixed.